internal/controllers/keypair/controller.go

```diff
@@ -33,8 +33,9 @@ import (
 
 const controllerName = "keypair"
 
-// +kubebuilder:rbac:groups=openstack.k-orc.cloud,resources=keypairs,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=openstack.k-orc.cloud,resources=keypairs/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=openstack.k-orc.cloud,resources=Keypairs,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=openstack.k-orc.cloud,resources=Keypairs/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch;update;patch
```

Comment:
In my understanding we shouldn't have to change the RBAC. Why was it necessary?

Reply:
I think it was me playing with the scaffolding getting a mixup and then having to rebase/cherry-pick is what created this mixture (the secret part is leftovers from having the server side key generation I probably missed)